login.html

```diff
@@ -3,7 +3,59 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>판다마켓 - Login</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="styles/account/account.css" />
+    <title>판다마켓 - 로그인</title>
+    <link rel="icon" href="/images/icon_favicon.ico" />
   </head>
-  <body></body>
+
+  <body>
+    <div class="loginContainer">
+      <header>
+        <a href="/">
+          <img
+            src="images/img_logo.png"
+            alt="판다마켓 로고"
+            class="image-logo"
+          />
+        </a>
+      </header>
+
+      <form>
+        <label>이메일 <input type="email" /></label>
```

Comment:
💊 제안
현재 코드에서는 태그와 태그가 들여쓰기 없이 한 줄로 붙어 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
        <label>
            이메일
            <input type="email" />
        </label>
```